classifier.py

```diff
@@ -264,7 +268,9 @@ def imshow(img, labels=None, pred=None, names=None, nmax=64, verbose=False, f=Pa
     ax = ax.ravel() if m > 1 else [ax]
     plt.subplots_adjust(wspace=0.05, hspace=0.05)
     for i in range(n):
-        ax[i].imshow(blocks[i].squeeze().permute((1, 2, 0)))  # cmap='gray'
+        im = blocks[i].squeeze().permute((1, 2, 0)).numpy()
+        img_n = cv2.normalize(src=im, dst=None, alpha=0, beta=255, norm_type=cv2.NORM_MINMAX, dtype=cv2.CV_8U)
+        ax[i].imshow(img_n)  # cmap='gray'
         ax[i].axis('off')
```

Comment:
@glenn-jocher FYI I've just pushed this for fixing the black images and clip warning.